db/seeds.rb

```diff
@@ -37,13 +37,13 @@
 
 order1 = Order.create(id: 3, name: "Colin", address: "400 Wash", city: "Denver", state: "CO", zip: 80203)
 
-ItemOrder.create(order_id: order1.id, item: pull_toy, quantity: 1, price: pull_toy.price)
-ItemOrder.create(order_id: order1.id, item: dog_bone, quantity: 2, price: dog_bone.price)
-ItemOrder.create(order_id: order1.id, item: tire, quantity: 3, price: tire.price)
-ItemOrder.create(order_id: order1.id, item: rim, quantity: 4, price: rim.price)
-ItemOrder.create(order_id: order1.id, item: chew_rope, quantity: 5, price: chew_rope.price)
-ItemOrder.create(order_id: order1.id, item: squeaky_toy, quantity: 6, price: squeaky_toy.price)
-ItemOrder.create(order_id: order1.id, item: kong_toy, quantity: 7, price: kong_toy.price)
-ItemOrder.create(order_id: order1.id, item: tire_parts, quantity: 8, price: tire_parts.price)
-ItemOrder.create(order_id: order1.id, item: axle, quantity: 9, price: axle.price)
-ItemOrder.create(order_id: order1.id, item: seat, quantity: 10, price: seat.price)
+order1.item_orders.create(order_id: order1.id, item: pull_toy, quantity: 1, price: pull_toy.price)
+order1.item_orders.create(order_id: order1.id, item: dog_bone, quantity: 2, price: dog_bone.price)
+order1.item_orders.create(order_id: order1.id, item: tire, quantity: 3, price: tire.price)
+order1.item_orders.create(order_id: order1.id, item: rim, quantity: 4, price: rim.price)
+order1.item_orders.create(order_id: order1.id, item: chew_rope, quantity: 5, price: chew_rope.price)
+order1.item_orders.create(order_id: order1.id, item: squeaky_toy, quantity: 6, price: squeaky_toy.price)
+order1.item_orders.create(order_id: order1.id, item: kong_toy, quantity: 7, price: kong_toy.price)
+order1.item_orders.create(order_id: order1.id, item: tire_parts, quantity: 8, price: tire_parts.price)
+order1.item_orders.create(order_id: order1.id, item: axle, quantity: 9, price: axle.price)
+order1.item_orders.create(order_id: order1.id, item: seat, quantity: 10, price: seat.price)
```

Comment:
Shouldn’t this be user.orders.create instead of order1.item_orders.create?

Reply:
Sorry I mean at line 38

Reply:
It's not creating an order off of the user - this is for the joins table `Item_orders` and the many to many relationship so we're setting each item_id to an order_id